src/test/java/com/google/devtools/build/lib/analysis/util/AnalysisTestCase.java

```diff
@@ -229,6 +230,9 @@ protected void useRuleClassProvider(ConfiguredRuleClassProvider ruleClassProvide
                     .add(
                         PrecomputedValue.injected(
                             ModuleFileFunction.REGISTRIES, ImmutableSet.of(registry.getUrl())))
+                    .add(
```

Comment:
this part should be unnecessary (analysisMock does the job already). `REGISTRIES` is different because we need to initialize `registry` first.